dev-tools/mage/integtest.go

```diff
@@ -319,6 +539,30 @@ func dockerComposeProjectName() string {
 	return projectName
 }
 
+// kindClusterName returns the cluster name to use with kind.
+// It is passed to kind. And is passed to our Go and Python testing libraries
+// through the KIND_CLUSTER_NAME environment variable.
+func kindClusterName() string {
+	commit, err := CommitHash()
+	if err != nil {
+		panic(errors.Wrap(err, "failed to construct kind cluster name"))
+	}
+
+	version, err := BeatQualifiedVersion()
+	if err != nil {
+		panic(errors.Wrap(err, "failed to construct kind cluster name"))
+	}
+	version = strings.NewReplacer(".", "_").Replace(version)
+
+	clusterName := "{{.BeatName}}_{{.Version}}_{{.ShortCommit}}-{{.StackEnvironment}}"
+	clusterName = MustExpand(clusterName, map[string]interface{}{
+		"StackEnvironment": StackEnvironment,
+		"ShortCommit":      commit[:10],
+		"Version":          version,
+	})
+	return clusterName
+}
```

Comment:
Move this to some kubernetes specific file?

Reply:
See `integtest_kubernetes.go`